src/libraries/System.Resources.Extensions/src/System/Resources/Extensions/DeserializingResourceReader.cs

```diff
@@ -3,19 +3,17 @@
 
 using System.ComponentModel;
 using System.IO;
+using System.Resources.Extensions.BinaryFormat;
 using System.Runtime.Serialization;
 using System.Runtime.Serialization.Formatters.Binary;
 
 namespace System.Resources.Extensions
 {
     public partial class DeserializingResourceReader
     {
-        private bool _assumeBinaryFormatter;
+        private static readonly bool UseBinaryFormatter = AppContext.TryGetSwitch("System.Resources.Extensions.UseBinaryFormatter", out bool fileConfig) && fileConfig;
```

Comment:
```suggestion
        private static readonly bool UseBinaryFormatter = AppContext.TryGetSwitch("System.Resources.Extensions.UseBinaryFormatter", out bool isEnabled) && isEnabled;
```
Nit: `fileConfig` for the local name does not make sense to me here/